src/web-auth/index.js

```diff
@@ -160,6 +161,31 @@ WebAuth.prototype.parseHash = function(options, cb) {
   ) {
     return cb(null, null);
   }
+  var responseType = (this.baseOptions.responseType || options.responseType || '').split(' ');
+  if (
+    responseType.length > 0 &&
```

Comment:
If you already have a passing indexOf check over the responseType string var, this one is useless.

Reply:
Even if this is an array, the check is already being done with indexOf. There's no need to add this size check (is redundant)

Reply:
I renamed to responseTypes to make it more clear